### ISSUE TITLE
Google My Business - Quota exceeded

### ISSUE THREAD

**Issue by brentadamson** [brentadamson/dataconnector#52]
Just setup Google My Business OAuth2 and doing a simple GET request to https://mybusinessaccountmanagement.googleapis.com/v1/accounts results in an error: `Quota exceeded for quota metric 'Requests per minute' of service 'mybusinessaccountmanagement.googleapis.com for consumer 'project_number:<number>'`

The [prerequisites](https://developers.google.com/my-business/content/prereqs) say to fill out a form, which I did. It might take a few days for them to review it then I'll try it again.

**Comment by brentadamson** [comment]
I received an email from them stating that third parties (like us) aren't being allowed to setup an OAuth2 client. I think the solution here is to make it so that people can use their own OAuth2 clients. It's something I've been wanting to add. Closing this in favor of 
 https://github.com/brentadamson/dataconnector/issues/53. Once that is complete, you'll be able to setup your own OAuth2 client in Google Cloud console (I can help) and then it's just a matter of entering your client id, secret and redirect url.
